app/src/main/java/com/parking_system_kotlin/mvp/view/ParkingReservationView.kt

```diff
@@ -0,0 +1,30 @@
+package com.parking_system_kotlin.mvp.view
+
+import android.app.Activity
+import com.parking_system_kotlin.fragments.DateReservationDialogFragment
+import com.parking_system_kotlin.listeners.ListenerDateTime
+import com.parking_system_kotlin.mvp.contracts.ParkingReservationContract
+import com.parking_system_kotlin.mvp.view.base.ActivityView
+import com.parking_system_kotlin.utils.Constants
+import com.parkingsystemkotlin.databinding.ActivityReservationBinding
+
+class ParkingReservationView(activity: Activity, private val binding: ActivityReservationBinding) : ActivityView(activity),
+    ParkingReservationContract.ParkingReservationView {
+
+    override fun showDatePicker(listenerDateTime: ListenerDateTime, dateSelector: Boolean) {
+        val reservationDate: DateReservationDialogFragment = DateReservationDialogFragment.newInstance(listenerDateTime, dateSelector)
+        fragmentManager?.let { reservationDate.show(it, Constants.DIALOG_FRAGMENT_ENTRY_DATE_PICKER) }
```

Comment:
if you just use it once, try to avoid creating additional vars

Reply:
done